core/web/router.go

```diff
@@ -290,6 +290,9 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		authv2.GET("/jobs/:ID/runs", paginatedRequest(prc.Index))
 		authv2.GET("/jobs/:ID/runs/:runID", prc.Show)
 		authv2.POST("/jobs/:ID/runs", prc.Create)
+
+		lgc := LogController{app}
+		authv2.PATCH("/log/debug/enable", lgc.ToggleDebug)
```

Comment:
JSONAPI prefers URLs to represent resources.

```suggestion
		authv2.PATCH("/log", lgc.ToggleDebug)
```